Core/HLE/sceIo.cpp

```diff
@@ -2184,8 +2185,7 @@ static u32 sceIoDread(int id, u32 dirent_addr) {
 		PSPFileInfo &info = dir->listing[dir->index];
 		__IoGetStat(&entry->d_stat, info);
 
-		strncpy(entry->d_name, info.name.c_str(), 256);
-		entry->d_name[255] = '\0';
+		truncate_cpy(entry->d_name, info.name.c_str());
```

Comment:
It's possible (though unlikely) that games may expect the zero-fill behavior of strncpy.

(and also for other PSP-visible structs - Module.)

-[Unknown]